students/ZackConnaughton/lessons/lesson09/assignment/database.py

```diff
@@ -0,0 +1,190 @@
+"""
+creates database with customer info and product info
+"""
+
+import argparse
+import csv
+import sys
+import logging
+import datetime
+from pymongo import MongoClient
+
+def disable_logging(funct):
+    def return_function(*args, **kwargs):
+        logger = logging.getLogger()
+        if ARG_PARSE.disable:
+            logger.disabled = True
+        funct(*args, **kwargs)
+        logger.disabled = False
+    return return_function
+
+
+def parse_cmd_arguments():
+    """
+    arg parser to determine if logging should be disabled from decorated functions
+    """
+    parser = argparse.ArgumentParser(description='Disable logging.')
+    parser.add_argument('-d', '--disable', help='disable logging for @disable_logging decorated functions', action='store_true')
```

Comment:
I think action='store_true' is not doing anything. 
Good solution to take advantage of argparse being global. Fortunately, we rarely have to do this sort of shenanigans. Your decorator should have a doc string. :)